src/pages/Filing/FilingApp/FileSubmission.tsx

```diff
@@ -237,40 +258,66 @@ export function FileSubmission(): JSX.Element {
-                            : // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-                              dataUpload || dataGetSubmissionLatest
-                              ? 'text-successColor'
-                              : 'text-[#0072CE]'
-                      }`}
-                      message={
-                        <span className='font-medium'>
-                          {isLoadingUpload
-                            ? 'Upload in progress'
-                            : errorUpload
-                              ? 'Upload failed'
-                              : // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-                                dataUpload || dataGetSubmissionLatest
-                                ? 'Upload complete'
-                                : ''}
-                        </span>
-                      }
+                      statusPriorityPipe={[
+                        { condition: isLoadingUpload, value: 'updating' },
+                        {
+                          condition:
+                            errorUpload ||
+                            dataGetSubmissionLatest?.state ===
+                              FileSubmissionState.UPLOAD_FAILED,
+                          value: 'error',
+                        },
+                        {
+                          condition: dataUpload || dataGetSubmissionLatest,
+                          value: 'approved',
+                        },
+                        { condition: true, value: '' }, // Default condition
+                      ]}
+                      classNamePriorityPipe={[
+                        {
+                          condition: isLoadingUpload,
+                          value: 'text-inProgressUploadValidation',
+                        },
+                        {
+                          condition:
+                            errorUpload ||
+                            dataGetSubmissionLatest?.state ===
+                              FileSubmissionState.UPLOAD_FAILED,
+                          value: 'text-errorColor',
+                        },
+                        {
+                          condition: dataUpload || dataGetSubmissionLatest,
+                          value: 'text-successColor',
+                        },
+                        { condition: true, value: 'text-[#0072CE]' }, // Default condition
+                      ]}
+                      messagePriorityPipe={[
+                        {
+                          condition: isLoadingUpload,
+                          value: 'Upload in progress',
+                        },
+                        {
+                          condition:
+                            errorUpload &&
+                            errorUpload.message ===
+                              FILE_SIZE_LIMIT_ERROR_MESSAGE,
+                          // TODO: Decide on error message
```

Comment:
@shindigira
All error handling should follow the standard template. The text next to the x will always describe the status of the upload (not started (I have seen no examples of this one yet), in progress, failed, complete) or validation (not started, in progress, failed, complete). 

For this instance it would be:

Status
x Upload failed
x Validation not started

We can include a more descriptive form-level alert for instances where we have specific information about what caused the error to occur. Is this error meant to capture the front end file size check? Please confirm and I will add this error to our content doc. We are capturing all messaging (success and error) both in the content doc and in the Figma file for the sake of consistency across instances.